test/distributed/test_composability.py

```diff
@@ -353,9 +345,9 @@ def apply_dp(partial_model):
 
         # Run the pipeline
         if pp_group.rank() == 0:
-            pipeline_schedule._step_microbatches(arg_mbs=input_mb)
```

Comment:
Nice thanks for removing! I checked the test folder, it doesn't look like any of our tests are using `_step_microbatches` anymore